app/views/devise/unlocks/new.html.erb

```diff
@@ -1,16 +1,12 @@
 <h2>Resend unlock instructions</h2>
-
 <%= form_for(resource, as: resource_name, url: unlock_path(resource_name), html: { method: :post }) do |f| %>
   <%= devise_error_messages! %>
-
   <div class="field">
     <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true %>
+    <%= f.email_field :email, required: true, autofocus: true %>
```

Comment:
we may want to reuse this `autofocus: true` in learnmode if it isnt already there